### PR TITLE
Porting guide restructured 3

### PR DESCRIPTION
**Pull request by abkro**
This is the reworked porting guide with sections added for Halium-9 porting.

### CODE REVIEW

File: porting/configure_test_fix/Sound.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    load-module module-droid-discover voice_virtual_stream=true
+
+and replace it with this::
+
+    load-module module-droid-discover rate=48000 quirks=+unload_call_exit
+
+At the end of the file, append this::
+
+    ### Automatically load the audioflinger glue
```
Comment (NotKit) [inline porting/configure_test_fix/Sound.rst:18]: This valid only for Halium 7 devices

File: porting/configure_test_fix/Sound.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The default configuration file used on Ubuntu Touch is ``touch.pa``. This file is located in the ``/etc/pulse`` directory on your device and it will need adjustment in order for sound to function properly. Extract the file and copy it to the ``ubuntu`` directory you created in your device repo (see :ref:`Overlay`).
+
+Locate the line::
+
+    load-module module-droid-discover voice_virtual_stream=true
+
+and replace it with this::
+
+    load-module module-droid-discover rate=48000 quirks=+unload_call_exit
```
Comment (NotKit) [inline porting/configure_test_fix/Sound.rst:14]: Isn't voice_virtual_stream=true needed for volume controls during calls? touch-android9.pa still has that. The whole change seems a bit device-specific to me, not something that should be done by default unless needed.

**Review by NotKit — APPROVED** [review]
I've left some comments, but otherwise since this is a large change set, I'd prefer to see it merged and then fix smaller pieces individually. Great job on doing all of this!

**Review by doniks — APPROVED** [review]
Thanks a lot @abkro !

**Pull request merged**